src/Symfony/Component/Cache/Adapter/PhpArrayAdapter.php

```diff
@@ -321,7 +321,7 @@ public function warmUp(array $values): array
 
         $dump .= "\n], [\n\n{$dumpedValues}\n]];\n";
 
-        $tmpFile = uniqid($this->file, true);
+        $tmpFile = tempnam(dirname($this->file), basename($this->file));
```

Comment:
Note the side effect, on the directory can not be written, file will be create in the system's temporary directory. This shouldn't happen, but that's not an issue.

Reply:
There is check before.